Kickstarter-iOS/DataSources/BackerDashboardPagesDataSource.swift

```diff
@@ -7,12 +7,10 @@ internal final class BackerDashboardPagesDataSource: NSObject, UIPageViewControl
 
   internal init(delegate: UIViewController, sort: DiscoveryParams.Sort) {
     let backedController = BackerDashboardProjectsViewController()
-    let savedController = BackerDashboardProjectsViewController()
+    backedController.configureWith(projectsType: .backed, sort: sort)
```

Comment:
true that we mostly use storyboards but don't think that's the reason we adopted that style. was more about getting a fully configured vc at once instead of instantiating it and then doing work on it